src/components/ResourceEdit/SelectNote.tsx

```diff
@@ -174,6 +183,10 @@ export const SelectNotes = (props: IProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [bookCd, findRef]
   );
+  useEffect(() => {
+    if (!refOverride)
+      setRefLevel(planType(plan)?.scripture ? RefLevel.Verse : RefLevel.All);
+  }, [refOverride, plan, planType]);
```

Comment:
I don't think it works to include planType here. In the past my experience is that when functions are included this will refresh too often. You could see if you observe the same thing by putting a console log in this useEffect. I am guessing you'll see it being called every time ... way more often than when plan or refOverride changes. Since planType is just a function of plan, it is only when plan changes that you actually want this useEffect to fire.